Lab10/src/main/java/org/example/entities/Abiturient.java

```diff
@@ -0,0 +1,97 @@
+package org.example.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.InputMismatchException;
+import java.util.Locale;
+import java.util.Scanner;
+
+@Entity
+@Table(name = "abiturients")
+@Setter
+@Getter
+public class Abiturient {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long ID;
+    @Column(name = "lastname")
+    private String lastName;
+    @Column(name = "firstname")
+    private String firstName;
+    @Column(name = "fathername")
+    private String fatherName;
```

Comment:
Keep using the column naming style as for `average_point` column and separate multiword columns by underscore
```suggestion
    @Column(name = "last_name")
    private String lastName;
    @Column(name = "first_name")
    private String firstName;
    @Column(name = "father_name")
    private String fatherName;
```

Reply:
Obviosly, a simple refactoring